app/config/config_dev.yml

```diff
@@ -43,3 +43,9 @@ monolog:
 
 #swiftmailer:
 #    delivery_address: me@example.com
+
+swp_bridge:
+    options:
+        curl: # http://guzzle.readthedocs.org/en/latest/faq.html#how-can-i-add-custom-curl-options
+            10203: # integer value of CURLOPT_RESOLVE
+                 - "example.com:5050:localhost"  # This will resolve the host example.com to your localhost
```

Comment:
let's got wih ```http://superdesk.dev:5050``` also here. and question is if we realy need this in config_dev.yml - as a developer i can easly create host like that. No port forwarding is needed